libs/language-server/src/stdlib/domain/material-science/MatSciDoiStandardizer.jv

```diff
@@ -0,0 +1,77 @@
+// Transformer to standardize DOI references and remove dx.doi.org/
```

Comment:
No real review from my side, but I noticed that you use the "simple" `//` comments here.

If you use `/** ... */` comments, the documentation can be picked up by Langium, so we can show it when hovering over a reference or on code completion :)

Reply:
Done, thanks :)